spec/README.md

```diff
@@ -84,7 +84,7 @@ A reference to a _term_ looks like this.
 Updates to this specification will not change
 the syntactical meaning, the runtime output, or other behaviour
 of valid messages written for earlier versions of this specification
-that only use functions and expression attributes defined in this specification.
+that only use functions defined in this specification.
```

Comment:
I think we'll want to keep this change, but reject the rest of this, since `attribute` is part of the syntax and we guarantee round trip with the data model (we do not claim that attributes in the data model do anything)

Reply:
#654 was the cherry-pick PR.